spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/SystemFunctionPushDownHelper.java

```diff
@@ -29,6 +31,21 @@ public static void createUnpartitionedTable(SparkSession spark, String tableName
     insertRecords(spark, tableName);
   }
 
+  public static void createUnpartitionedTable(
+      SparkSession spark, String tableName, Map<String, String> props) {
+    if (props.isEmpty()) {
+      createUnpartitionedTable(spark, tableName);
+    } else {
+      String tblProps = Joiner.on("','").withKeyValueSeparator("'='").join(props);
+      sql(
+          spark,
+          "CREATE TABLE %s (id BIGINT, ts TIMESTAMP, data STRING) USING iceberg TBLPROPERTIES ('%s')",
```

Comment:
Isn't this SQL invalid if there are no table properties because it would result in `TBLPROPERTIES ('')`?

Reply:
There is a branch checking:
```java
    if (props.isEmpty()) {
      createUnpartitionedTable(spark, tableName);
    } else {
      String propsString = Joiner.on("','").withKeyValueSeparator("'='").join(props);
      sql(
          spark,
          "CREATE TABLE %s (id BIGINT, ts TIMESTAMP, data STRING) USING iceberg TBLPROPERTIES ('%s')",
          tableName,
          propsString);
      insertRecords(spark, tableName);
```